src/build.d

```diff
@@ -525,21 +525,13 @@ alias style = makeRule!((builder, rule)
         .msg("(GIT) DScanner")
         .target(dscannerDir)
         .condition(() => !exists(dscannerDir))
-        // .command([
-        //     // FIXME: Omitted --shallow-submodules because  it errors for libdparse
-        //     env["GIT"], "clone", "--depth=1", "--recurse-submodules", "--branch=v0.8.0",
-        //     "https://github.com/dlang-community/Dscanner", dscannerDir
-        // ])
-        .commandFunction(()
-        {
-            const git = env["GIT"];
-            run([git, "clone", "--depth=15", "--recurse-submodules", "--branch=v0.8.0",
-                "https://github.com/dlang-community/Dscanner", dscannerDir]);
-
-            // TODO: Remove this temporary fix for the invalid error messages in
-            // backend/ptrtab.d when D-Scanner upgrades DParse
-            run([git, "-C", dscannerDir.buildPath("libdparse"), "checkout", "63559db5cc6fa38c01bdda36e09638b5f20fb8e5"]);
-        })
+        .command([
+            // FIXME: Omitted --shallow-submodules because it requires a more recent
+            //        git version which is not available on buildkite
```

Comment:
This comment seems out of date.

Reply:
Last run from BuildKite:

```
[...]
Command: git clone --depth=1 --recurse-submodules --shallow-submodules --branch=v0.9.0-beta.1 https://github.com/dlang-community/Dscanner /var/lib/buildkite-agent/builds/ci-agent-1b8888d4-05cf-49df-8be7-5111d465b14e-1/dlang/dmd/dmd/generated/linux/release/64/dscanner
-------------------------------------------------------------
ERROR: error: unknown option `shallow-submodules'
usage: git clone [<options>] [--] <repo> [<dir>]
[...]
```

Reply:
Ah its not passing.

Reply:
For reference, `--shallow-submodules` is available since `git v2.9.0` which was released in July 2016.